extra_data/write_cxi.py

```diff
@@ -163,7 +163,26 @@ def collect_data(self):
 
         return layouts
 
-    def write(self, filename):
+    @staticmethod
+    def _fill_value(path, layout, fillvalues):
+        array_type = layout[path].dtype.type(0).item()
```

Comment:
I think the type check can be a bit simpler:

```python
np.issubdtype(dt, np.floating)
# Or
dt.kind == 'f'
````

Reply:
Thanks, I removed it completely since now all 3 arrays have different defaults

Reply:
oh, I forgot that raw `data` is integer :'D